packages/ui/Marquee/MarqueeBase.js

```diff
@@ -182,6 +191,8 @@ const MarqueeBase = kind({
 	},
 
 	computed: {
+		'aria-label': ({'aria-label': aria, children}) => aria == null &&
+			typeof children === 'string' ? children : aria,
```

Comment:
I slightly more robust version of this would map over the children and join all children. This would be necessary when evaluating content within children:

```
// children.length would be 3 but all are strings
this {count === 0 ? 'does not have' : 'has'} children
```

https://codesandbox.io/s/inspiring-williams-j3rmv

Reply:
I briefly considered this, but do you have any examples of this pattern?  If there's one, I would definitely suggest we robustify this.  Otherwise, I think you could get carried away trying to iterate through everything.  On the other hand, is potentially omitting some data and having it repeated once better than repeating the full string twice?

Reply:
I don't have product app examples but I think the one above is a reasonable pattern and likely how I'd have written the code.

Reply:
Do you have a response for the last question there?

Reply:
Updated and now will only apply label if absolutely necessary.